drape_frontend/frontend_renderer.cpp

```diff
@@ -1433,7 +1433,7 @@ void FrontendRenderer::RenderScene(ScreenBase const & modelView, bool activeFram
     Render2dLayer(modelView);
     RenderUserMarksLayer(modelView, DepthLayer::UserLineLayer);
 
-    if (m_buildingsFramebuffer->IsSupported())
+    if (m_buildingsFramebuffer->IsSupported() && !m_routeRenderer->IsHelicopterRoute())
```

Comment:
Don't understand why it is here?
The difference here is only in call order and supporting Framebuffers

Reply:
The only difference in this if-else block is order of drawing. And to draw route on top of 3D buildings I force `else` block for helicopter route type.
Is there any better solution?

Reply:
Well, if we want a route line to be on top of everything (other routes don't overlap buildings?), it is an acceptable solution for now.